README.md

```diff
@@ -53,19 +53,21 @@ You can also open the project by double clicking on Simplenote.xcworkspace file,
 
 ## Setup Credentials
 
-Simplenote is powered by the [Simperium Sync'ing protocol](https://www.simperium.com). We distribute **testing credentials** that help us authenticate your application, and verify that the API calls being made are valid.
+Simplenote is powered by the [Simperium Sync'ing protocol](https://www.simperium.com). We distribute **testing credentials** that help us authenticate your application, and verify that the API calls being made are valid. Once the Simperium account is created, you can register an app and access the APP ID and necessary API keys.
 
 **⚠️ Please note → We're not accepting any new Simperium accounts at this time.**
 
-After you've created your own Simperium application, copy the sample config. For example:
+After you've created your own Simperium application, run the following command in the terminal in order to copy the sample config:
 
 ```
 mkdir -p ~/.configure/simplenote-ios/secrets && cp Simplenote/SPCredentials-demo.swift ~/.configure/simplenote-ios/secrets/SPCredentials.swift
 ```
 
-Then edit the new `Simplenote/Credentials/SPCredentials.swift` file and change the `simperiumAppID` and `simperiumApiKey` fields to the correct values for your new app.
+This will copy the demo SPCredentials file into the correct directory with the basic details for an OSS contributor. Then edit the new `Simplenote/Credentials/SPCredentials.swift` file and change the `simperiumAppID` and `simperiumApiKey` fields to the correct values that appear in your Simperium app. 
 
-This will allow you to compile and run the app on a device or a simulator.
+If the build fails with an "Account Creation Failed" error and Xcode asks for your GitHub credentials most likely you need to update the Swift Packages before being able to build the app. Package issues need to be resolved before you can build the app, so the credentials file is put in place. For this you can setup GitHub under Xcode > Preferences > Accounts, then enter your account and access token. If you have 2FA on your GitHub account you won't be able to connect to without the SSH key.
```

Comment:
I think it might be helpful here to add a link to the GH instructions for how to get the access token and how to setup the ssh key

Reply:
```suggestion
If the build fails with an "Account Creation Failed" error and Xcode asks for your GitHub credentials most likely you need to update the Swift Packages before being able to build the app. Package issues need to be resolved before you can build the app, so the credentials file is put in place. For this you can setup GitHub under Xcode > Preferences > Accounts, then enter your account and [access token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token). If you have 2FA on your GitHub account you won't be able to connect to without the [SSH key.](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
```

Reply:
Those are great additions 👍 , thanks!